### DISCUSSION TITLE
Feature to search in just one site?

### DISCUSSION THREAD

**Discussion by ccvision** [microsoft-search/pnp-modern-search#1796] [Ideas]
I'm well aware that this is already possible with a WebID: or Path: query,
but it'd be easier if this was just a checkbox or something along those lines.

Could this be implemented?

**Comment by wobba** [comment]
It's a good idea and we will have to think about how this would play with what you enter into the query template. I don't think we aspire to have the same "easy" UX as the highlighted content web part, because any logic will have to work across both the Graph provider and the SharePoint provider.
